.github/workflows/run-e2e-tests.yml

```diff
@@ -0,0 +1,47 @@
+name: Run end-to-end tests
+
+on:
+  workflow_dispatch:
+
+  push:
+    branches: [master]
+    paths: # add other modules when there are under e2e tests
```

Comment:
Can we add `powertools-parameters` now so that it'll pick up the tests i've added as well once these both merge 👼 ? I don't think a glob on a directory that doesn't exist should break